user-service/src/main/java/uk/gov/gchq/palisade/service/user/model/UserResponse.java

```diff
@@ -32,11 +32,11 @@
 
 
 /**
- * This data represents the output for user-service which will include the User identified by the service.
- * This will be forwarded to the resource-service to identify the Resources associated with this query.
+ * This data represents the output for User Service which will include the User identified by the service.
```

Comment:
reword a bit. 
Something like, if a valid user is found, a UserResponse object is created, containing the User retireved from the backing store, and is sent to the Resouce Service to find resources associated with the request query

Reply:
The user isn't always going to be acquired from the backing store. It could be returned by an external system.